docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -74,3 +74,12 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3.
 {{% /admonition %}}
+
+
+## Permission validation on custom role creation and update
+
+<!-- Mihaly Gyongyosi -->
+
+_Generally available in Grafana Enterprise_
+
+With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during a custom role creation or update are validated against the list of [available permissions and their scopes]({{< relref "../administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions" >}}). If the request contains a permission that is not available or the scope of the permission is not valid, the request will be rejected with an error message.  
```

Comment:
You did the right thing with the relref, we're just updating to a simplified linking format.
```suggestion
With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during a custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.  
```